src/commands/start.ts

```
@@ -150,6 +151,12 @@ export default class Start extends Command {
 
           resolve();
         });
+
+        cmd.on('error', (e) => {
+          this.log(_error(`Error: spawning architect-${service_config.language}-launcher`));
+          this.log(_error(e.toString()));
+          this.exit(1);
```

We should bubble up this error in the promise rejection otherwise we have hanging promises.

---

So instead of calling exit:
`reject(error)`?

---

Yup. We can catch errors at the top and log them.